docs/whats_new/v0.4.1.md

```diff
@@ -1,9 +1,192 @@
+
+- Fixed error raising for setting key that does not exist in the configuration (#1613) @sxjscience
+
+- Fixed warning message about bf16. (#1625) @sxjscience
+
+- Fixed the corner case of calculating the gradient accumulation step (#1633) @sxjscience
+
+- Fixes for top-k averaging in the multi-gpu setting (#1707) @zhiqiangdon
+
+## Tabular
+
+- Limited RF `max_leaf_nodes` to 15000 (previously uncapped) (#1717) @Innixma
+  - Previously, for very large datasets RF/XT memory and disk usage would quickly become unreasonable. This ensures that at a certain point RF and XT will no longer become larger given more rows of training data. Benchmark results showed that the change is an improvement, particularly for the `high_quality` preset.
+
+- Limit KNN to 32 CPUs to avoid OpenBLAS error (#1722) @Innixma
+  - Issue  #1020. When training K-nearest-neighbors (KNN) models, sometimes a rare error can occur that crashes the entire process:
+
+    ```
+    BLAS : Program is Terminated. Because you tried to allocate too many memory regions.
+    Segmentation fault: 11
+    ```
+
+    It has so far only occurred on machines with >300 GB of memory (2 confirmed instances). In both cases, the system was not low on memory and had plenty to complete the task, yet the error still occurred.
+
+- Improved memory warning thresholds (#1626) @Innixma
+
+- Added `get_results` and `model_base_kwargs` (#1618) @Innixma
+  - Added `get_results` to searchers, useful for debugging and for future extensions to HPO functionality.
+    Added new way to init a `BaggedEnsembleModel` that avoids having to init the base model prior to initing the bagged ensemble model.
+
+- Update resource logic in models (#1689) @Innixma
+  - Previous implementation would crash if user specified `auto` for resources, fixed in this PR.
+  - Added `get_minimum_resources` to explicitly define minimum resource requirements within a method.
+
+- Updated feature importance default `subsample_size` 1000 -> 5000, `num_shuffle_sets 3` -> 5 (#1708) @Innixma
+  - This will improve the quality of the feature importance values by default, especially the 99% confidence bounds. The change increases the time taken by ~8x, but this is acceptable because of the numerous inference speed optimizations are done since these defaults were first introduced.
+
+- Added notice to ensure serializable custom metrics (#1705) @Innixma
+
+### Bug fixes
+
+- Fixed `evaluate` when `weight_evaluation=True` (#1612) @Innixma
+  - Previously, AutoGluon would crash if the user specified `predictor.evaluate(...)` or `predictor.evaluate_predictions(...)` when `self.weight_evaluation==True`.
+
+- Fixed RuntimeError: dictionary changed size during iteration (#1684, #1685) @leandroimail
+
+- Fixed CatBoost custom metric & F1 support (#1690) @Innixma
+
+- Fixed repeated bagging failing to fit repeats beyond the first when time limit is specified due to `self.model_base` being `None` in the loaded model (#1702) @Innixma
```

Comment:
Can remove this line, it is a fix for a bug that was introduced after v0.4 was released

Reply:
removed